docs/developer/developer-setup.md

```diff
@@ -190,15 +186,15 @@ Following services are dockerized:
 - Minio Database (external document storage)
 
 If the script is started with `--slim`, only the Blockchain, API, Provisioning and Frontend will be started.
-For further details how to use the script, print out its help section with `sh scripts/development/start-dev.sh -h`.
+For further details how to use the script, print out its help section with `bash scripts/development/start-dev.sh --help`.
 
 It takes some time to build and run at the first launch. After that, source codes of all services are live reloaded. That means any change in `./src` folder is reflected in respective container automatically.
 
 The frontend should be availaible as usual at http://localhost:3000
 
 Docker Compose ensures that services are communicating and have correct environment variables set. Docker Compose puts all services in the same network and exposes needed ports.
 
-You can inspect each container individually: `docker logs --follow CONTAINER`. Where _CONTAINER_ represents selected value of NAMES column container in output of `docker ps` command.
+You can inspect each container individually: `docker logs --follow CONTAINER`. Where _CONTAINER_ represents name of the service that can be found in the `.yml` file.
 
 :::note
 Hot reloading with docker requires a lot of processing power (due to docker volumes). If you experience huge delays while hot-reloading, you should start the services you need by yourself without docker.
```

Comment:
Dockerized Application (recommended) guide ends here. Make clear, that the following **is not** needed if using docker.

Reply:
done